docs/wiki/configurator/failsafe-tab.md

```diff
@@ -4,8 +4,125 @@ sidebar_position: 5
+| **Ascent rate (meter/second)**          | The rate at which the craft will ascend to the necessary altitude.                                                                                                                                                                                                                                                                                                                                                                                |
+| **Return ground speed (meters/second)** | The speed at which the craft will return to the home position.                                                                                                                                                                                                                                                                                                                                                                                    |
+| **Maximum pitch angle**                 | The maximum pitch angle the craft will use to return to the home position. Rescue throttle usually needs to be increased if the pitch angle is increased. If pitch angle is too low, the craft might not be able to return in high winds.                                                                                                                                                                                                         |
+| **Descent rate (meters/second)**        | The rate at which the craft will descend to the ground when it reaches the home position.                                                                                                                                                                                                                                                                                                                                                         |
+| **Throttle minimum**                    | The **minimum** throttle value the craft will use to return to the home position.                                                                                                                                                                                                                                                                                                                                                                 |
+| **Throttle maximum**                    | The **maximum** throttle value the craft will use to return to the home position.                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Make sure that the craft has enough throttle and pitch angle achieve the set return velocity into any expected headwind!